include/openssl/ssl.h

```diff
@@ -387,6 +387,14 @@ typedef int (*SSL_verify_cb)(int preverify_ok, X509_STORE_CTX *x509_ctx);
         SSL_OP_NO_TLSv1|SSL_OP_NO_TLSv1_1|SSL_OP_NO_TLSv1_2|SSL_OP_NO_TLSv1_3)
 # define SSL_OP_NO_DTLS_MASK (SSL_OP_NO_DTLSv1|SSL_OP_NO_DTLSv1_2)
 
+/*
+ * When using DTLS/SCTP, include the terminating zero in the label
+ * used for computing the endpoint-pair shared secret. Required for
+ * interoperability with implementations having this bug like older
+ * version of OpenSSL.
+ */
+# define SSL_OP_DTLS_SCTP_LABEL_LENGTH_BUG               0x20000000U
```

Comment:
This clashes with SSL_OP_NO_TLSv1_3. That's problematic if we intend to backport this to 1.1.1, because we've run out of numbers in 1.1.1. In 3.0.0 (i.e. current master) it's not a problem because we can reuse one of the values marked as "Reserved until 1.2.0"

Reply:
Can you suggest a better value? Porting to to 1.1.1 makes sense from my point of view. I can suggest a CHANGES entry...

Reply:
Maybe we can reuse the value of a constant which only applies to TLS and not DTLS? The use of `SSL_OP_DTLS_SCTP_LABEL_LENGTH_BUG` is DTLS over SCTP specific...

Reply:
https://github.com/openssl/openssl/pull/7957/commits/a7fb64a36bee8bbe242e487c218ce04a5606483b changes `SSL_OP_DTLS_SCTP_LABEL_LENGTH_BUG` to `SSL_MODE_DTLS_SCTP_LABEL_LENGTH_BUG`.